docs/xks/developer-guide/scheduling-scaling.md

```diff
@@ -0,0 +1,280 @@
+
+The [Horizontal Pod Autoscaler](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/) can do this automatically. The version `autoscaling/v1` of the Horizontal Pod Autoscaler only supports scaling based on CPU utilization. Future versions will support scaling based on more metrics types.
+
+```yaml
+apiVersion: autoscaling/v1
+kind: HorizontalPodAutoscaler
+metadata:
+  name: app
+spec:
+  minReplicas: 1
+  maxReplicas: 5
+  targetCPUUtilizationPercentage: 60
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: app
+```
+
+> The Pod Disruption Budget discusses the problems related to setting a minimum availability which is the same or higher than the static replica count. The same problem can occur with Horizontal Pod Autoscalers if the value is smaller than the minimum replica value.
+
+For more details refer to the [Horizontal Pod Autoscaler walkthrough](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/).
+
+<!-- Section about custom metrics -->
+
+## Priority Class
+
+The Kubernetes scheduler will out of the box treat each Pod with the same priority. The scheduler will assign Pods to Nodes in the order in which they were created, when multiple Pods without a set priority are waiting to be scheduled. This is usually not an issue as Pods tend to be assigned to Nodes quickly. However them
+scheduling duration may increase if multiple Horizontal Pod Autoscalers were to increase the replica count at the same time, as new Nodes would have to be provisioned first. In this case the queue would grow while waiting for more capacity in the cluster. Some applications may be more critical than others for the survival
+of a product. Waiting for the applications turn may not be the optimal solution if other applications have no problem waiting a bit longer to start running.
+
+Setting Priority Class to a Pod can help the scheduler decide which Pods are more important and should be assigned to a Node first. In XKS there are three Priority Classes available.
+
+* `tenant-low`
+* `tenant-medium`
+* `tenant-high`
+
+The Priority Class is set with the `priorityClassName` field. Always start off with using the Priority Class `tenant-low`, and decide later on if it should be changed. This is for the simple reason that if every Pod has a high priority no Pod will be considered of high priority as it would be the same as setting no priority

```

Comment:
Should you mention that it's okay to not define a priorityClassName?

Reply:
We want people to set priority class names so I would skip this. Might be good to add a mutation for this in the future.